Fastfile

```diff
@@ -110,7 +110,8 @@ end
 
 def _build_ipa
   update_app_identifier(xcodeproj: ENV['FL_UPDATE_PLIST_PROJECT_PATH'],
-                        plist_path: ENV['FL_UPDATE_PLIST_PATH'])
+                        plist_path: ENV['FL_UPDATE_PLIST_PATH'],
+                        app_identifier: ENV['FL_UPDATE_PLIST_APP_IDENTIFIER'])
```

Comment:
my memory is failing me… why are any of these needed? aren't they the defaults if nothing is provided in these parameters?

Reply:
good question... i'll take a look at the fastlane source code (although right now my build is failing because the app_identifier is not provided)

Reply:
ah OK, got it. it's because this function uses different env vars as defaults. we want to override with the env vars used by the `update_info_plist` function, to avoid duplication.